### PR TITLE
add: volume command

### PR DESCRIPTION
**Pull request by iceisblue**
Add the volume command, allowing users to see or change the volume.

### CODE REVIEW

File: src/commands/voice/volume.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  arguments: [
+    { name: "subcommand", type: "subcommand", required: false }
+  ],
+  guildOnly: true,
+  execute(message) {
+    const player = bot.lavadenoManager.players.get(message.guildId.toString());
+
+    if (!player) return message.reply("No player in this guild.");
+    
+    return message.reply(`Current volume is ${player?.volume}.`);
```
Comment (itohatweb) [inline src/commands/voice/volume.ts:18]: seems it can be undefined so add a fallback?

File: src/commands/voice/volume.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  },
+});
+
+createSubcommand("volume", {
+  name: "set",
+  description: "Set the volume of the player.",
+  aliases: ["s", "change"],
+  permissionLevels: [PermissionLevels.MODERATOR],
+  arguments: [
+    { name: "value", type: "number", minimum: -Infinity, required: true }
```
Comment (itohatweb) [inline src/commands/voice/volume.ts:28]: why `-Infinity`? shouldn't this be 0?

File: src/commands/voice/volume.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  ] as const,
+  guildOnly: true,
+  execute: (message, args) => {
+    const player = bot.lavadenoManager.players.get(message.guildId.toString());
+    if (!player) return message.reply("No player in this guild.");
+
+    if (args.value > 100 || args.value < 1) return message.reply("Please provide valid volume from 1 to 100.");
```
Comment (Yaikava) [inline src/commands/voice/volume.ts:35]: The values are wrong. Lavadeno takes 0-1000.
So either change it here or multiply it by 10 in `player.setVolume()`

File: src/commands/voice/volume.ts (hunk 2 above)
Comment (iceisblue) [reply src/commands/voice/volume.ts:28]: This may be 0 or -Infinity. Doesn't really matter.

File: src/commands/voice/volume.ts (hunk 3 above)
Comment (iceisblue) [reply src/commands/voice/volume.ts:35]: Right, forgot about that.

File: src/commands/voice/volume.ts (hunk 2 above)
Comment (itohatweb) [reply src/commands/voice/volume.ts:28]: Why does it not matter? 0 !== -Infinity. If you check it later in code you could remove it I think.

File: src/commands/voice/volume.ts (hunk 1 above)
Comment (Yaikava) [reply src/commands/voice/volume.ts:18]: `.volume` cannot be undefined as long as player exists.
So just `player.volume` would do.

Comment (iceisblue) [reply src/commands/voice/volume.ts:18]: I don't really know if this can be undefined. VSCode just autocompleted that for me lol

Comment (Yaikava) [reply src/commands/voice/volume.ts:18]: It cannot, it defaults to `100` on creation.

File: src/commands/voice/volume.ts (hunk 2 above)
Comment (iceisblue) [reply src/commands/voice/volume.ts]: When I tested it, when I use -Infinity then it return the "Please provide..." on - numbers. When I have 0, it doesn't. But this isn't really important, I don't think people will try to set it on -

File: src/commands/voice/volume.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  description: "See the current volume.",
+  aliases: ["vol", "v"],
+  arguments: [
+    { name: "subcommand", type: "subcommand", required: false }
+  ],
+  guildOnly: true,
+  execute(message) {
+    const player = bot.lavadenoManager.players.get(message.guildId.toString());
+
+    if (!player || typeof player === undefined) return message.reply("No player in this guild.");
```
Comment (Yaikava) [inline src/commands/voice/volume.ts:16]: I think here you can just do this.
```ts
if (!player)
```

Comment (iceisblue) [reply src/commands/voice/volume.ts]: Yeah, that was just to test something.

**Review by itohatweb — APPROVED** [review]

**Pull request merged**